src/components/Share/PetCardItem.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from "next/image";
+import { z } from "zod";
+import Button from "./Button";
+import { IconPetGender } from "@/icons/IconPetGender";
+import { cn } from "@/utils/helpers";
+
+const petCardItemSchema = z.object({
+  name: z.string(),
+  gender: z.string(),
+  image: z.string().url(),
+});
```

Comment:
Gender can either be male or female, so you can modify the schema:
`const petCardItemSchema = z.object({
  name: z.string(),
  gender: z.enum(["female", "male"]),
  image: z.string().url(),
});`

Reply:
fixed! i add PerProps in Types in utils